src/julia.h

```diff
@@ -746,7 +746,13 @@ JL_DLLEXPORT int64_t jl_gc_total_bytes(void);
 JL_DLLEXPORT uint64_t jl_gc_total_hrtime(void);
 JL_DLLEXPORT int64_t jl_gc_diff_total_bytes(void);
 
-JL_DLLEXPORT void jl_gc_collect(int);
+typedef enum {
+    heuristic = 0,      // use heuristics to determine the collection type
+    full = 1,           // force a full collection
+    incremental = 2,    // force an incremental collection
+} jl_gc_collection_t;
```

Comment:
I think C enum is not namespaced so you certainly want the `jl_` prefix here. Also I think constants usually use all capital.